integrations/cdi/jedis-cdi/README.adoc

```diff
@@ -79,12 +79,87 @@ redis.clients.jedis.JedisPool.orders.port
+
+Any documentation for these properties that exists may be found in the
+javadocs for the
+https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPool.html[`JedisPool`]
+and
+https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/util/Pool.html[`Pool`]
+classes.
+
+For advanced configuration use cases, it is also possible to configure
+the
+https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPoolConfig.html[`JedisPoolConfig`]
+object used by the Helidon Jedis CDI Integration project to create
+`JedisPool` instances.  In this case,
+https://static.javadoc.io/org.eclipse.microprofile.config/microprofile-config-api/1.3/org/eclipse/microprofile/config/Config.html#getPropertyNames--[property
+names] that start with
+`redis.clients.jedis.JedisPoolConfig.`_instanceName_`.` will be
+parsed, and the remaining portion of each such name will be treated as
+a Java Bean or constructor property of `JedisPool` (or of its
+superclasses).
+
+Accordingly, the `JedisPoolConfig` Java Bean properties that can be
+set are as follows, where `instanceName` should be replaced with the
+actual named used in application code:
+
+* `redis.clients.jedis.JedisPoolConfig.instanceName.blockWhenExhausted`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.evictionPolicyClassName`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.fairness`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.jmxEnabled`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.jmxNameBase`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.jmxNamePrefix`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.lifo`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.maxIdle`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.maxTotal`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.maxWaitMillis`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.minEvictableTimeMillis`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.minIdle`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.numTestsPerEvictionRun`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.softMinEvictableIdleTimeMillis`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.testOnBorrow`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.testOnCreate`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.testOnReturn`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.testWhileIdle`
+* `redis.clients.jedis.JedisPoolConfig.instanceName.timeBetweenEvictionRunsMillis`
+
+Any documentation for these properties that exists may be found in the
+javadocs for the
+https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPoolConfig.html[`JedisPoolConfig`],
+https://commons.apache.org/proper/commons-pool/apidocs/org/apache/commons/pool2/impl/GenericObjectPoolConfig.html[`GenericObjectPoolConfig`]
+and
+https://commons.apache.org/proper/commons-pool/apidocs/org/apache/commons/pool2/impl/BaseObjectPoolConfig.html[`BaseObjectPoolConfig`]
+classes.
```

Comment:
For more information about these properties, see the following Javadocs:
- https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPoolConfig.html[`JedisPoolConfig`]
- https://commons.apache.org/proper/commons-pool/apidocs/org/apache/commons/pool2/impl/GenericObjectPoolConfig.html[`GenericObjectPoolConfig`]
- https://commons.apache.org/proper/commons-pool/apidocs/org/apache/commons/pool2/impl/BaseObjectPoolConfig.html[`BaseObjectPoolConfig`]